modules/drivers/snowflake/test/metabase/test/data/snowflake.clj

```diff
@@ -112,8 +112,10 @@
 
 (defmethod tx/destroy-db! :snowflake
   [_ {:keys [database-name]}]
-  (let [database-name (qualified-db-name database-name)]
-    (jdbc/execute! (no-db-connection-spec) [(format "DROP DATABASE \"%s\";" database-name)])
+  (let [database-name (qualified-db-name database-name)
+        sql           (format "DROP DATABASE \"%s\";" database-name)]
+    (println "[Snowflake]" sql)
```

Comment:
intended?

Reply:
Actually, yeah. It's in the test code that loads data into Snowflake IF it's not already loaded. Other SQL statements in that namespace are printed in a similar way. This one was missing it tho